src/client/dfs/dfs.c

```diff
@@ -884,6 +906,7 @@ open_file(dfs_t *dfs, daos_handle_t th, dfs_obj_t *parent, int flags,
 	if (rc != 0) {
 		D_ERROR("daos_array_open_with_attr() failed, "DF_RC"\n",
 			DP_RC(rc));
+		D_GOTO(out, rc = daos_der2errno(rc));
 		return daos_der2errno(rc);
```

Comment:
Unreachable code?  I'm surprised this got past the compiler to be honest unless I'm misreading it.

Reply:
yea that's a merge conflict that i didn't resolve properly. sigh